src/Resources/config/doctrine/model/Redirect.orm.xml

```diff
@@ -9,10 +9,6 @@
             <generator strategy="AUTO"/>
         </id>
 
-        <unique-constraints>
-            <unique-constraint columns="source"/>
```

Comment:
Should probably be `source, channel_id`

Reply:
Yes, but is that possible? It's a many to many relationship